<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>440</width>
    <height>520</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Sign Up</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color:rgb(13, 13, 13);</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>40</y>
     <width>201</width>
     <height>61</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 170, 255);
font: 34pt &quot;MS Reference Sans Serif&quot;;</string>
   </property>
   <property name="text">
    <string>SIGN UP</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>150</y>
     <width>101</width>
     <height>45</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color:rgb(255, 226, 0); font: 17pt &quot;Swis721 BlkCn BT&quot;;</string>
   </property>
   <property name="text">
    <string>Email:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="Email">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>150</y>
     <width>241</width>
     <height>45</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color:rgb(255, 255, 255);
font: 63 15pt &quot;Segoe UI Semibold&quot;;</string>
   </property>
   <property name="cursorPosition">
    <number>0</number>
   </property>
  </widget>
  <widget class="QLineEdit" name="Password">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>240</y>
     <width>241</width>
     <height>45</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color:rgb(255, 255, 255);
font: 63 15pt &quot;Segoe UI Semibold&quot;;</string>
   </property>
   <property name="echoMode">
    <enum>QLineEdit::Password</enum>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>240</y>
     <width>101</width>
     <height>45</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color:rgb(255, 226, 0); font: 17pt &quot;Swis721 BlkCn BT&quot;;</string>
   </property>
   <property name="text">
    <string>Password:</string>
   </property>
  </widget>
  <widget class="QPushButton" name="SignUpBtn">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>440</y>
     <width>361</width>
     <height>45</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(170, 255, 255);</string>
   </property>
   <property name="text">
    <string>SIGN UP</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="RemeberMeFlag">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>410</y>
     <width>101</width>
     <height>20</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string>remember me</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="ShowPasswordFlag">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>300</y>
     <width>101</width>
     <height>20</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string>show password</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
